config.default.yaml

```diff
@@ -7,6 +7,13 @@ database:
   username: mongoadmin
   password: mongoadminsecret
   srv: false
+api:
+  auth:
+    secret_key: "1234" # Secret key for JWT authentication, change it to a strong secret
+    token_expiration: 0 # jwt token expiration in seconds, 0 to disable
```

Comment:
May want to add a note here for anyone deploying a BOOM system that this is not recommended as the default for security concerns. Or just make the default 7 days or something, and in our own dev configs we can set to zero if desired.